source/_docs/installation.markdown

```diff
@@ -10,6 +10,10 @@ footer: true
 redirect_from: /getting-started/installation/
 ---
 
+<p class='note'>
+Note: Beginners should check our [Getting started guide](/getting-started/) first. This is for users that require advanced installtions
```

Comment:
Remove `Note:`, it wil be rendered as a note. Also, there is a typo in `installtions` and the period at the end is missing.